bundle/manifests/siteconfig.clusterserviceversion.yaml

```diff
@@ -85,15 +85,15 @@ metadata:
                         "ipv4": {
                           "address": [
                             {
-                              "ip": "10.16.231.3",
+                              "ip": "1.2.3.4",
```

Comment:
I mean even these should probably be using the reserved IP range for docs/examples.
This comment applies to all IP addresses in this example text
https://en.wikipedia.org/wiki/Reserved_IP_addresses

Reply:
Great suggestion! Have updated the IP{v4,v6} addresses and MAC addresses as well!